files/perfectSquare.py

```diff
@@ -0,0 +1,38 @@
+# Your name here:

```

Comment:
This template should go on the Csci 127 repo, please remove from here and make a pull request to add it to that repo. Thank you!